src/constants/Messages.js

```diff
@@ -143,7 +143,7 @@ const welcomeQuestions = [
     'If you could remove anything from the game, what would it be?',
     'If you could add anything to the game, what would it be?',
     'What are your favorite games?',
-    'Ask jwiggs any question you want.',
+    'What would you like to ask Jwiggs?',
```

Comment:
Since we're changing this, it might make more sence to do this
```suggestion
    'What would you like to ask Jwiggs (the developer)?',
```

Reply:
Also should remove the swear warnings as they are no longer used.